test/_regtest_outputs/test_optimal_schedulers.test_regression[stems-5].out

```diff
@@ -1,8 +1,8 @@
-2,0.133914,45,45,4,3,400
-2,0.119788,45,45,3,5,200
-2,0.076407,33,33,3,5,400
-2,0.164278,45,45,3,5,500
-2,0.096557,33,33,5,2,300
+2,0.133914,45,51,4,3,400
```

Comment:
Unsure how the changes resulted in this change, but the F1 didn't change I let it pass